understanding/20/resize-text.html

```diff
@@ -131,7 +134,7 @@ <h2>Examples of Resize text</h2>
          </li>
          
          <li>
-            A user uses a zoom function in his user agent to change the scale of the content.
+            A user uses a full-page zoom function in their user agent to change the scale of the content.
```

Comment:
This is personal bias. I get jarred every time I see a user/their combination. I intentionally rewrite these as users/their. 
I realize that with 3rd-person pronoun preferences, "they" is becoming viewed as singular, but always found it awkward notwithstanding, and believe making it agree 'traditionally' by usage is never wrong, and arguably desirable for clarity in some situations.

Totally understand my views on subject/pronoun agreement will be viewed as quaint/old-fashioned/pitiable by some...

Reply:
while yes, this view does seem a bit old to me... we can move to plural.

though looking at this closer, the "Users use" pattern always grates me. How would we feel about "People use..." ?

Reply:
decided to turn this list item around (also to avoid the "user uses ... user agent"